rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Rubric.java

```diff
@@ -72,6 +72,9 @@ public class Rubric implements PersistableEntity<Long>, Serializable, Cloneable
 
     private Boolean weighted = Boolean.FALSE;
 
+    @Column(nullable = false)
+    private Boolean draft = Boolean.FALSE;
```

Comment:
Just added `@Column(nullable = false)`, so the conversion script shouldn't be needed

Reply:
The script is needed anyways, there could be institutions with auto.ddl = false.

Thanks! Please pair a conversion script for Sakai 23.

Reply:
oups, ready 👍